geommethods.yaml

```diff
@@ -154,6 +154,10 @@
 - name: Length
   comment: returns g's length
   type: float64Property
+- name: MakeValid
+  comment: Repair an invalid geometry, returning a valid output.
+  type: unary
+  geosFunction: GEOSMakeValid_r
```

Comment:
`geosFunction` should not be needed. It's only needed if the GEOS function does not match the name of the Go function. In this case, they're identical.